assets/js/broken_redirect_list.js

```diff
@@ -2331,11 +2330,11 @@ validurls['/docs/developer_guide/platforms/web/changelog'] = '/docs/developer_gu
 validurls['/docs/developer_guide/platforms/xamarin/changelog'] = '/docs/developer_guide/changelogs/?sdktab=xamarin';
 
 validurls['/docs/developer_guide/platforms/web/google_tag_manager'] = '/docs/developer_guide/sdk_integration/?sdktab=web#web_google-tag-manager';
-validurls['docs/developer_guide/platforms/fireos/push_notifications'] = '/docs/developer_guide/push_notifications/?sdktab=android';
-validurls['docs/developer_guide/platforms/fireos/android_13'] = 'docs/developer_guide/platforms/fireos/android_13';
-validurls['docs/developer_guide/platforms/fireos/google_play_privacy'] = 'docs/developer_guide/platforms/fireos/google_play_privacy';
-validurls['docs/developer_guide/platforms/fireos/sample_apps'] = 'docs/developer_guide/platforms/fireos/sample_apps';
-validurls['docs/developer_guide/platforms/fireos/changelog'] = 'docs/developer_guide/platforms/fireos/changelog';
+validurls['/docs/developer_guide/platforms/fireos/push_notifications'] = '/docs/developer_guide/push_notifications/?sdktab=android';
+validurls['/docs/developer_guide/platforms/fireos/android_13'] = '/docs/developer_guide/platforms/android/android_13';
+validurls['/docs/developer_guide/platforms/fireos/google_play_privacy'] = '/docs/developer_guide/platforms/fireos/google_play_privacy';
+validurls['/docs/developer_guide/platforms/fireos/sample_apps'] = '/docs/developer_guide/platforms/fireos/sample_apps';
+validurls['/docs/developer_guide/platforms/fireos/changelog'] = '/docs/developer_guide/platforms/fireos/changelog';
```

Comment:
```suggestion
validurls['/docs/developer_guide/platforms/fireos/sample_apps'] = '/docs/developer_guide/platforms/fireos/sample_apps';
validurls['/docs/developer_guide/platforms/fireos/changelog'] = '/docs/developer_guide/platforms/fireos/changelog';
```
These also go through 10 redirects before hitting a 404.